mini-pools-ios/Models/ContributionModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ContributionModel.swift
+//  mini-pools-ios
+//
+//  Created by Kaplún, Sebi on 12/20/17.
+//  Copyright © 2017 Kaplún, Sebi. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
```

Comment:
Swift 4 has a native way to decode JSONs. `SwiftyJSON` is still easier though ;)
https://benscheirman.com/2017/06/swift-json/

Reply:
👍  good to know